shippable.yml

```diff
@@ -42,8 +42,6 @@ matrix:
     - env: T=windows/2012-R2/4
     - env: T=windows/2016/4
 
-    - env: T=network
-
```

Comment:
Lets keep the VyOS testing on Shippable for a while so we have some overlap until we're confident in the results we're getting by running things under Zuul.